luna_pinyin.dict.yaml

```diff
@@ -48923,6 +48923,9 @@ use_preset_vocabulary: true
 竍	shi
 俧	zhi
 
+#Unicode 13.0
```

Comment:
no need for a separate paragraph

Reply:
Alright, I saw the heading above for Unicode 7.0 and it seemed weird to just tack it on to that, but if that's okay I'll do that. :+ )